samples/client/petstore/swift5/alamofireLibrary/docs/FakeAPI.md

```diff
@@ -130,7 +130,7 @@ Test serialization of outer number types
 // The following code samples are still beta. For any issue, please report via http://github.com/OpenAPITools/openapi-generator/issues/new
 import PetstoreClient
 
-let body = 987 // Double | Input number as post body (optional)
+let body = Data([9, 8, 7]) // Double | Input number as post body (optional)
```

Comment:
@aymanbagabas Data is being applies here but it shouldn't.
It seams that `Data` is reaplacing `Double`, `Int`, `Int64`, `Float`, and maybe others